### ISSUE TITLE
Feature request: offline mode

### ISSUE THREAD

**Issue by Fjollete** [Mosc/Glider#230]
I'd like the ability to toggle an offline mode for those times when you know you'll be without connection (such as aboard a plane) or in a place with spotty connection. 

The feature would store/cache a user-defined number of posts and their comments so that you could keep using the app offline.

**Comment by bethropolis** [comment]
yes please, this is the only feature keeping me from using Glider, I'd really like an offline support as I mostly like reading and catching up when I'm mostly offline.
